doc/build-windows.md

```diff
@@ -81,9 +81,26 @@ The first step is to install the mingw-w64 cross-compilation tool chain:
 
     sudo apt install g++-mingw-w64-x86-64
 
-Ubuntu Bionic 18.04 <sup>[1](#footnote1)</sup>:
+Next, set the default `mingw32 g++` compiler option to POSIX (For Ubuntu Bionic 18.04 and up<sup>[1](#footnote1)</sup>):
```

Comment:
Why mention "18.04 and up"? If this is Ubuntu-specific, I don't think anyone use pre-xenial distros.

Reply:
true ... I guess that part can be dropped, and the note is improved by just saying "this is the next step"

Reply:
addressed in dafab2b